map_model/src/pathfind/driving.rs

```diff
@@ -10,6 +10,7 @@ use abstutil::MultiMap;
 
```

Comment:
minor nit: Apparently this is used by bikes too? Can we rename this module to `pathfind::vehicle`?

Reply:
Sounds good, I'll do that in a followup.

Relatedly, I need to rename `CarID` to `VehicleID` and `Bus*` (ID, Stop, Route...) to `PublicTransit*` (buses or trains).